blocks/header/header.js

```diff
@@ -212,27 +219,43 @@ export default async function decorate(block) {
     decorateIcons(nav);
 
     // Add search form
-    const searchIcon = navSearch.children[0].children[0].children[0];
+    const searchIcon = navSearch.querySelector('.icon');
+    const searchTarget = navSearch.querySelector('a[href]')?.getAttribute('href') || '/search';
     const searchForm = document.createElement('form');
     searchForm.classList.add('search-form');
-    searchForm.setAttribute('action', '/search');
-    navSearch.replaceChild(searchForm, navSearch.children[0]);
-    const searchElement = [...navSearch.children][0];
-    searchElement.innerHTML = `
-      <input type="text" class="search-input" id="search" value="" placeholder="${window.placeholders?.default?.search || 'Search'}" />
-      <button type="submit" class="search-submit"></button>
-    `;
-    const button = searchElement.children[1];
-    button.appendChild(searchIcon);
-    searchElement.addEventListener('submit', (e) => {
+    searchForm.setAttribute('action', searchTarget);
+    const searchInput = document.createElement('input');
+    searchInput.type = 'text';
```

Comment:
we have the opportunity to change the type to search instead because it adds an "X" icon that removes the typed text

Reply:
cool, did not know about this one, will change it